### PR TITLE
Fix dividerbar height issue (#170)

### PR DESCRIPTION
**Pull request by adidesh20**
Fixes Issue #170, dividerbar height now changes in response to scrolling on the right tab section.

### CODE REVIEW

**Review by jzzheng22 — APPROVED** [review]
LGTM

File: src/Renderer/UI/MainView.fs [hunk 1]
```diff
@@ -147,6 +148,14 @@ let inline setDragMode (modeIsOn:bool) (model:Model) dispatch =
...
 let dividerbar (model:Model) dispatch =
     let isDraggable = model.RightPaneTabVisible = WaveSim
+    let heightAttr = 
+        // On app startup, the RightSection is not rendered, so using scrollHeight
+        // results in an error. Check if project is opened to avoid this.
+        match model.CurrentProj with
+        | None -> Height "100%"
+        | Some _ -> 
+            let rightSection = document.getElementById "RightSection"
+            Height rightSection.scrollHeight
```
Comment (jzzheng22) [inline src/Renderer/UI/MainView.fs:158]:
```suggestion
    let heightAttr = 
        let rightSection = document.getElementById "RightSection"
        if (isNull rightSection) then Height "100%"
        else Height rightSection.scrollHeight
```

**Review by jzzheng22 — CHANGES_REQUESTED** [review]
On second thoughts: see if this works?

FYI `isNull` is located in `JSHelpers.fs`

File: src/Renderer/UI/MainView.fs (hunk 1 above)
Comment (jzzheng22) [reply src/Renderer/UI/MainView.fs:158]: I haven't tested the above code, might need to add an `open JSHelpers`.

Comment (jzzheng22) [reply src/Renderer/UI/MainView.fs:158]: Example usage in `Sheet.fs` and `WaveformSimulationView.fs`

**Review by jzzheng22 — APPROVED** [review]

**Comment by adidesh20** [comment]
I have also changed the cursor for the dividerbar from grab to ew-resize as it makes more sense.

**Pull request merged**